test/integration/util/util.go

```diff
@@ -69,12 +68,8 @@ func StartScheduler(clientSet clientset.Interface) (factory.Configurator, Shutdo
 	stopCh := make(chan struct{})
 	schedulerConfigurator := createSchedulerConfigurator(clientSet, informerFactory, stopCh)
 
-	sched, err := scheduler.NewFromConfigurator(schedulerConfigurator, func(conf *factory.Config) {
-		conf.Recorder = evtBroadcaster.NewRecorder(legacyscheme.Scheme, v1.EventSource{Component: "scheduler"})
-	})
-	if err != nil {
-		klog.Fatalf("Error creating scheduler: %v", err)
-	}
+	config, _ := schedulerConfigurator.CreateFromConfig(schedulerapi.Policy{})
```

Comment:
should we be discarding the error here?

Reply:
No, we need to check this error, I have changed it. 
Thanks for your reminder and review.